src/examples/websocket_client.c

```diff
@@ -45,22 +45,23 @@ parse_cli_args(int argc, char * const *argv) {
   }
 }
 
-void websocket_ready_handler(mtev_websocket_client_t *client) {
-  mtev_websocket_client_send(client, 0x1, "Hello world!", 13);
+mtev_boolean websocket_ready_handler(mtev_websocket_client_t *client, void *closure) {
+  return mtev_websocket_client_send(client, 0x1, "Hello world!", 13);
 }
 
-int websocket_msg_handler(mtev_websocket_client_t *client, int opcode,
-                          const unsigned char *msg, size_t msg_len) {
+mtev_boolean websocket_msg_handler(mtev_websocket_client_t *client, int opcode,
+                                   const unsigned char *msg, size_t msg_len,
+                                   void *closure) {
   char buf[256];
```

Comment:
This is example code but it will stack overflow if someone sends a msg_len > 256